sdk/ml/azure-ai-ml/azure/ai/ml/constants/_job/distillation.py

```diff
@@ -0,0 +1,16 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+
+class DataGenerationTaskType:
+    NLI = "NLI"
+    NLU_QA = "NLU_QA"
+    CONVERSATION = "CONVERSATION"
+    MATH = "MATH"
+    SUMMARIZATION = "SUMMARIZATION"
+
+
+class DataGenerationType:
+    LabelGeneration = "label_generation"
```

Comment:
constants should be all uppercase to follow python naming convention
the actual values can be left as lowercase

Reply:
Done